swh/config.py

```
@@ -0,0 +1,119 @@
+
+
+class DatabaseSettings(BaseSettings):
+    """PostGIS connection settings, populated from POSTGRES_* env vars."""
+
+    model_config = SettingsConfigDict(env_prefix="POSTGRES_")
+
+    host: str = "postgis"
+    port: int = 5432
+    db: str = "gis"
+    user: str = "socialwarehouse"
+    password: str = "CHANGEME"
```

Ruff (0.15.6)

[error] Possible hardcoded password assigned to: "password"

(S105)